uiowa/uiowa
# I2522 grad scholar ct

Closes #2552 

Also removing the book module from the the `site_grad_uiowa_edu` split, since it is not going to be used.

# How to test

1. blt ds --site=grad.uiowa.edu
4. add a person -> select 'Scholar'
5. Make sure fields: First name, last name, credentials, title/position, department, email, phone, SROP session, education, bio, research areas, website, research project title, mentor url, abstract fields are visible
    - SROP session & mentor links- more than one will be separated by comma
6. add content to fields and make sure it displays. When go to people page make sure teaser is visible. Then click on CT and make sure full view is visible as well
7. Make sure each SROP session is clickable and will take you to teaser view of all people w/ same SROP session year
